site/css/typography/_fonts.css

```diff
@@ -152,4 +163,14 @@
     font-size: 40px;
     line-height: 1.1em;
   }
+
+  .fontM2 {
+    font-size: 40px;
+    line-height: 1.1em;
+  }
```

Comment:
Changing fontM2 effects lots of other pages. Just double checking that this is correct?